src/hammer.mlg

```diff
@@ -791,18 +816,25 @@ let hammer_hook_tac prefix name =
       Msg.notice ("Warning: " ^ s);
       ltac_apply "idtac" []
   end
+}
 
 TACTIC EXTEND Hammer_hook_tac
-| [ "hammer_hook" string(prefix) string(name) ] -> [ hammer_hook_tac prefix name ]
+| [ "hammer_hook" string(prefix) string(name) ] -> { hammer_hook_tac prefix name }
 END
 
+{
 let pr_taclist _ _ _ lst = Pp.pr_comma () (* TODO: LC: I haven't figured out how to print a tactic *)
 
-ARGUMENT EXTEND taclist TYPED AS tactic_list PRINTED BY pr_taclist
-| [ tactic3(tac) "|" taclist(l) ] -> [ tac :: l ]
-| [ tactic3(tac) ] -> [ [ tac ] ]
+let wit_tactic_list = Genarg.ListArg wit_tactic
+
+}
+
+ARGUMENT EXTEND taclist TYPED AS tactic_list PRINTED BY { pr_taclist }
```

Comment:
You should use `tactic list` instead of defining the genarg by hand, it'll work.

Reply:
Done thanks.